filebeat/tests/files/logs/es_happy_startup.log

```diff
@@ -0,0 +1,55 @@
+{"type": "main", "timestamp": "2019-01-10T10:38:14,990+0100", "level": "INFO", "component": "o.e.e.NodeEnvironment", "cluster.name": "distribution_run", "node.name": "node-0",  "message": "using [1] data paths, mounts [[/ (/dev/disk1s1)]], net usable_space [187gb], net total_space [465.6gb], types [apfs]"  }
+{"type": "main", "timestamp": "2019-01-10T10:38:14,993+0100", "level": "INFO", "component": "o.e.e.NodeEnvironment", "cluster.name": "distribution_run", "node.name": "node-0",  "message": "heap size [989.8mb], compressed ordinary object pointers [true]"  }
+{"type": "main", "timestamp": "2019-01-10T10:38:14,996+0100", "level": "INFO", "component": "o.e.n.Node", "cluster.name": "distribution_run", "node.name": "node-0",  "message": "node name [node-0], node ID [BboLkxHbQAW5IyqyqAfBLA]"  }
```

Comment:
It looks like the `node.id` is known here (`BboLkxHbQAW5IyqyqAfBLA`). Any reason it couldn't be extracted into a JSON property?

Reply:
good spot, but I don't think that with current ES infrastructure code I can access that value earlier. I need to wait for a internal clusterStateUpdate to pass that value to log4j code
Will confirm that